src/PointwiseFunctions/Hydro/EquationsOfState/HybridEos.cpp

```diff
@@ -140,10 +140,11 @@ double HybridEos<ColdEquationOfState>::specific_internal_energy_upper_bound(
   // that the pressure is bounded by the total energy density,
   // i.e. p < e = rho * (1 + eps)
   if (ColdEquationOfState::is_relativistic and thermal_adiabatic_index_ > 2.0) {
-    return get(cold_eos_.specific_internal_energy_from_density(
-        Scalar<double>{rest_mass_density}));
```

Comment:
I'm not sure how we get the cold internal energy as an _upper bound_ for the hot internal energy?